watcher/scripts/watchFt.ts

```diff
@@ -0,0 +1,54 @@
+// This file is for when I (@bingyuyap) needs to seed the database manually for testing
```

Comment:
Since this is a public repo, I would maybe word it like this:
```suggestion
// This script can be used to seed the database manually for testing
```